modules/efk-logging-management-state-changing.adoc

```diff
@@ -19,9 +19,32 @@ and upgrades when in unmanaged state.
 An unmanaged deployment does not receive updates until the cluster logging is placed back into a managed state.
 ====
 
+.Prerequisite
+
+* The Cluster Logging Operator must be installed.
+
+* Have the name of the custom logging CR:
++
+----
+$ oc get clusterloggingoc get -n openshift-logging ClusterLogging
```

Comment:
`oc get -n openshift-logging ClusterLogging`

Reply:
Just saw that too!